ecs-init/config/common.go

```diff
@@ -47,8 +47,10 @@ const (
 	DefaultRegionName = endpoints.UsEast1RegionID
 
 	// cgroupMountpointEnv is the Environment Variable used to provide
-	// an alternative path to the cgroup mount on the host.
-	cgroupMountpointEnv = "CGROUP_MOUNTPOINT"
```

Comment:
While I doubt the number is very high, there is potential for customers to be using this old environment variable. I think we need to maintain `CGROUP_MOUNTPONT` as an option too.